ui/components/app/assets/nfts/nfts-items/nfts-items.js

```diff
@@ -157,7 +157,7 @@ export default function NftsItems({
 
   const updateNftDropDownStateKey = (key, isExpanded) => {
     const newCurrentAccountState = {
-      ...nftsDropdownState[selectedAddress][chainId],
+      ...nftsDropdownState?.[selectedAddress]?.[chainId],
```

Comment:
Do we know whether its expected that this can be undefined / how to reproduce it?

Reply:
I was only able to repro the error by forcing selectedAddress to be undefined;